### PR TITLE
Align files

### PR DESCRIPTION
**Pull request by architectbot**
This PR is generated by https://github.com/giantswarm/github

### CODE REVIEW

**Review by kubasobon — APPROVED** [review]

File: integration/test/crmapping/crmapping_test.go [hunk 1]
```diff
@@ -229,8 +229,8 @@ func ensureCRsExist(t *testing.T, client client.Client, inputFiles []string) {
 
 		if o.GetObjectKind().GroupVersionKind().Kind == "NamespaceList" {
 			lst := o.(*corev1.NamespaceList)
-			for _, i := range lst.Items {
-				err = client.Create(context.Background(), &i)
+			for i := range lst.Items {
```
Comment (whites11) [inline integration/test/crmapping/crmapping_test.go:232]: why do we need this change? Actually why do we need all these changes?

Comment (fiunchinho) [reply integration/test/crmapping/crmapping_test.go:232]: I guess they are linter recommendations

Comment (kubasobon) [reply integration/test/crmapping/crmapping_test.go:232]: `golangci-lint` has changed with architect-orb and it's raising errors on **go-build**:
```
service/controller/cluster/handler/clusterownerreference/resource.go:187:57: G601: Implicit memory aliasing in for loop. (gosec)
		err = controllerutil.SetControllerReference(&cluster, &machinePool, r.scheme)
```
https://app.circleci.com/pipelines/github/giantswarm/azure-operator/4330/workflows/d98f096b-84a2-46e7-8c3b-dad73762642f/jobs/21294/parallel-runs/0/steps/0-116

Comment (kubasobon) [reply integration/test/crmapping/crmapping_test.go:232]: If there's a better way of doing this, then I'll happily change my approach.

**Comment by kubasobon** [comment]
Shit. Auto-merge already merged this :/ I'll revert or make any necessary changes.

**Comment by whites11** [comment]
> Shit. Auto-merge already merged this :/ I'll revert or make any necessary changes.

don't bother I guess it's ok

**Pull request merged**